scripts/windows/configs/prep_omnibus.bat

```diff
@@ -1,6 +1,6 @@
 REM Prepare ruby default gemset to handle metasploit omnibus build
 git clone https://github.com/rapid7/metasploit-omnibus
 cd metasploit-omnibus
-gem install bundler -v2.1.4 --no-document && bundle install --binstubs
+gem install bundler -v2.2.3 --no-document && bundle install && bundle binstubs --all
```

Comment:
`bundle install --binstubs` is deprecated, so updating this to use the newer convention